django_mongodb_backend/lookups.py

```diff
@@ -34,8 +34,6 @@ def field_resolve_expression_parameter(self, compiler, connection, sql, param):
 
 
 def in_(self, compiler, connection):
-    if isinstance(self.lhs, MultiColSource):
-        raise NotImplementedError("MultiColSource is not supported.")
```

Comment:
Is this full feature support a consequence of a feature enhancement, or is it due to a change required from the 5.2 upgrade? If it's the former, could it be split into a different PR?

Reply:
I'm going to go with [the latter](https://github.com/django/django/pull/18404) and also [the import can be removed](https://github.com/mongodb/django-mongodb-backend/blob/43f477cc2e1a0981e53bc2a60ee0e049939c716f/django_mongodb_backend/lookups.py#L2).

Reply:
Essentially, `MultiColSource` was refactored to `ColPairs`. It's still unsupported but the error is raised differently.